packages/mint-css/theme/variables/index.css

```diff
@@ -0,0 +1,71 @@
+html.color-theme-in-transition,
+html.color-theme-in-transition *,
+html.color-theme-in-transition *::before,
+html.color-theme-in-transition *::after {
+  transition: all 300ms !important;
+  transition-delay: 0 !important;
+}
+
+html {
+  /* global colors */
+  --black: #121212;
+  --white: #ffffff;
+  --gray900: #44475b;
+  --gray800: #696c7c;
+  --gray700: #7c7e8c;
+  --gray600: #8f919d;
+  --gray500: #a1a3ad;
+  --gray400: #b0b2ba;
+  --gray300: #c7c8ce;
+  --gray200: #ddeee1;
+  --gray150: #e9e9eb;
+  --gray100: #f0f0f2;
+  --gray50: #f8f8f8;
+  --green500: #00B386;
+  --green300: #66e3c4;
+  --green100: #EBF9F5;
+  --purple500: #5367ff;
+  --purple300: #84a4ff;
+  --purple100: #eef0ff;
+  --yellow500: #ffb61b;
+  --yellow100: #fff5e0;
+  --red500: #eb5b3c;
+  --red100: #fae9e5;
+  --tempBg: #ffffff;
+  --tempTransparent:  rgba(0, 0, 0, 0);
```

Comment:
can add comment here why this is used

Reply:
@shubhamgavali1998 I missed adding the comment here, The same comment line is to be added, as I added in scripts.

Reply:
Done